src/test/scala-2/chiselTests/experimental/hierarchy/SeparateElaborationSpec.scala

```diff
@@ -142,6 +142,10 @@ class SeparateElaborationSpec extends AnyFunSpec with Matchers with Utils with T
         args = Array("-td", testDir, "--full-stacktrace", "--target", "chirrtl"),
         annotations = Seq(ChiselGeneratorAnnotation(() => new Testbench(dutDef)), ImportDefinitionAnnotation(dutDef))
       )
+
+      val tbFir = Source.fromFile(s"${testDir}/Testbench.fir").getLines().mkString
+      assert("extmodule AddOne".r.findAllMatchIn(tbFir).length == 1)
+      assert("""inst \S+ of AddOne""".r.findAllMatchIn(tbFir).length == 2)
```

Comment:
I would prefer we do this with FileCheck that that involves upgrading this entire Spec--LGTM because this is in the same style as the rest of the file.